change/@fluentui-react-2020-05-05-09-14-40-feat-react-theme-provider.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "react-theme-provider: Adding a universal ThemeProvider package which will provide universal css variable based themeing to converged components.",
```

Comment:
should be type none and pls fix comment to describe change correctly

Reply:
If I do none, it will not publish an initial publish.

Reply:
this is changefile for `fluent/react` package. in fact you marked the themeprovider package to be `private`. i thought that was intentional?

Reply:
huh... that's odd. good catch; digging.

Reply:
Straightened out. :) Thanks for calling that out! Great catch. I didn't intend to privatize the package, and would rather just start publishing it off the bat. Enables codesandbox experiments.